bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/guide/OptimizableGuideProbeGroup.java

```diff
@@ -20,13 +20,16 @@
      * @param guideStar the new guide star to incorporate in the target
      * environment
      *
+     * @param isBAGS determines if the guide star was chosen by background
+     * AGS (true) or manually selected (false)*
+     *
      * @param ctx the context of the observation, which contains information
      * needed to assign a guider to the new guide star
      *
      * @return a newly optimized TargetEnvironment which incorporates the given
      * <code>guideStar</code>
      */
-    TargetEnvironment add(SPTarget guideStar, ObsContext ctx);
+    TargetEnvironment add(SPTarget guideStar, boolean isBAGS, ObsContext ctx);
```

Comment:
Perhapsthe `isBAGS` boolean could be made a type to make it a stronger assertion